tests/test_buffer.py

```diff
@@ -1940,6 +1940,81 @@ def test_div_type(type):
         ):
             sarr2.idiv(sarr1)
 
+    def test_eye(self):
+        """Test eye() static method for creating identity matrices"""
+        eye2 = modmesh.SimpleArrayFloat64.eye(10)
+        for i in range(10):
+            for j in range(10):
+                if i == j:
+                    self.assertEqual(eye2[i, j], 1.0)
+                else:
+                    self.assertEqual(eye2[i, j], 0.0)
+
+    def test_scaled_eye(self):
+        """Test scaled_eye() static method for creating scaled identity
+        matrices"""
+        scaled_eye = modmesh.SimpleArrayFloat64.scaled_eye(10, 3.5)
+        for i in range(10):
+            for j in range(10):
+                if i == j:
+                    self.assertEqual(scaled_eye[i, j], 3.5)
+                else:
+                    self.assertEqual(scaled_eye[i, j], 0.0)
+
+    def test_hermitian(self):
+        """Test hermitian() method for creating conjugate transpose"""
+        ndarr = np.array([[1.0 + 2.0j, 3.0 + 4.0j, 5.0 + 6.0j],
+                          [7.0 + 8.0j, 9.0 + 10.0j, 11.0 + 12.0j],
+                          [13.0 + 14.0j, 15.0 + 16.0j, 17.0 + 18.0j]],
+                         dtype='complex128')
+
+        sarr = modmesh.SimpleArrayComplex128(array=ndarr)
+        shermitian = sarr.hermitian()
+        ndhermitian = ndarr.conj().T
+        print(shermitian.ndarray)
+        print(ndhermitian)
+        self.assertEqual(shermitian.shape, ndhermitian.shape)
+        np.testing.assert_equal(shermitian.ndarray, ndhermitian)
+
+    def test_hermitian_error(self):
```

Comment:
Consider to merge the error checking into the main test function as a section.  It looks like over-engineering to split at this early stage.